src/commands/profiles/list.js

```diff
@@ -13,19 +13,19 @@ class ProfilesList extends BaseCommand {
         accountSid: ENVIRONMENT_ACCOUNT_SID,
         region: ENVIRONMENT_REGION,
       };
-      this.userConfig.profiles.unshift(strippedEnvProfile);
```

Comment:
We should update the comment at [line 8](https://github.com/twilio/twilio-cli/blob/main/src/commands/profiles/list.js#L8) (userConfig.profiles) also to userConfig.projects.

Reply:
Thanks for pointing! Changed.